### DISCUSSION TITLE
Milvus 2.3 helm chart

### DISCUSSION THREAD

**Discussion by lyajedi** [milvus-io/milvus#26607] [Q&A and General discussion]
The helm chart doesn't include any hint for GPU resources allocation, which components should be allocated with GPUs for Milvus 2.3 cluster? Index node and Query node?

**Comment by xiaofan-luan** [comment]
We are working on the GPU documents, stay tuned

**Reply by lyajedi** [reply]
any ETA for that?

**Reply by lyajedi** [reply]
Hi, any news for this? or anything that I can refer to?

**Reply by xiaofan-luan** [reply]
So far GPU is only supported for standalone, helm install is still under development

https://milvus.io/docs/install_standalone-gpu-docker.md#Install-Milvus-Standalone-with-Docker-Compose-GPU

**Reply by mrceresa** [reply]
Hello, 
I would also be very interested in using GPU acceleration on milvus cluster.

Is there any update on how to configure it with helm or operator?

Thanks and regards
